_includes/date.html

```diff
@@ -0,0 +1 @@
+{{ article.date | date: "%-d %b %Y" }}
```

Comment:
this only works for things called ```article``` 

A better way is to do
```
{% assign my_date = include.date %}
{{ my_date | date: .....}}
```

and then to call
```
{% include date.html date=article.date %}
```

Reply:
Thanks for the pointer, incorporated.